env.config.sh

```diff
@@ -42,6 +42,7 @@ export SONAR_MYSQL_DATABASE="sonar"
 export SONAR_ACCOUNT_LOGIN="jenkins"
 export SONAR_DB_LOGIN=${SONAR_MYSQL_USER}
 export SONAR_DB_PASSWORD=${SONAR_MYSQL_PASSWORD}
+export CARTRIDGE_SOURCES="https://raw.githubusercontent.com/DavideVi/adop-cartridges/master/cartridges.yml"
```

Comment:
Once the repository has been forked, you just need to update the value here to reference Accenture.